vm/src/stdlib/io.rs

```diff
@@ -2688,13 +2738,18 @@ mod _io {
                         PyStr::from(self.slice()).into_ref(vm)
                     }
                 }
+                fn utf8_len(&self) -> Utf8size {
```

Comment:
i think this name is confusing due to `char::len_utf8()`

Reply:
Yeah, but I think this a decent name - gets the length in units of utf8 codepoints and utf8 bytes.

Reply:
At the very least I can't think of an alternative